test/fluree/db/query/history_test.clj

```diff
@@ -604,30 +614,126 @@
                                                         :ex/y "bar-cat"}
                                                 {:message "meow"})
             loaded-ledger (test-utils/retry-load conn ledger-name 100)]
-        (is (= [#:f{:assert  [{:ex/x "foo-3"
+        (is (pred-match?
+             [#:f{:assert  [{:ex/x "foo-3"
```

Comment:
Did you mean for this test not to use any of the pred matching? I would like to keep at least one actual did-the-hashes-change type test around, so I'm fine with that.

Reply:
No I meant to update it. Thanks!